Bugfix/cannot import gedcom file or add media files

tobikster opened the pull request:
Application targeting the Android 13 (API 33) can't use the `READ_EXTERNAL_STORAGE` permission anymore.
Instead, the more detailed permissions should be used.

Fix: Use correct permissions on API below and above 33.

Refers: #71 

Tested on:
- Pixel 3 (API 31)
- Android emulator (API 33)

michelesalvador commented:
Some minor remarks:

* Often you used tabular indentation instead of 4 spaces
* The logical order should be `IMAGES` then `VIDEO` then `AUDIO` in AndroidManifest, MediaFoldersActivity and F
* IMHO `checkMultiplePermissions` method should be below the comment `// Methods for image acquisition:`

Could you please fix these?

Personally I prefer one single commit for such a small bug fix to be included in master. But maybe this can be done with a squash merge.

tobikster commented:
I corrected the indentations.
I would suggest to commit Code Style configuration into the VCS, so everyone working at the project will have the same settings.
This should help with avoiding such differences in the future 🙂 

I changed the order of the permissions in the `AndroidManifest` file.

I didn't move the `checkMultiplePermissions` method below the comment — it isn't related to the image acquisition, but it could be used to check if the app can use any permissions.

michelesalvador commented:
@tobikster Can you please reorder permissions also in MediaFoldersActivity and F?

I have included `.idea/codeStyles/` into the VCS.

hannesa2 commented:
I expect, that now all files have code format issues 😀

You should do this, to format all at once
<img width="488" alt="image" src="https://user-images.githubusercontent.com/3314607/207154473-105cecbc-4568-4f39-92e9-51ed63876cf9.png">

hannesa2 commented:
Or better do this format all step _after_ this merge. Doing it this way, you avoid annoying fixes
